src/App.js

```diff
@@ -1,12 +1,23 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './App.scss';
+import Layout from './components/Layout';
 import Calculator from './components/Calculator';
 import Quote from './components/Quote';
+import Home from './components/Home';
 
 function App() {
   return (
     <div>
-      <Calculator />
-      <Quote />
+      <BrowserRouter>
+        <Layout />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/home" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="quote" element={<Quote />} />
+          <Route path="*" element={<div>Not Found 404</div>} />
+        </Routes>
+      </BrowserRouter>
```

Comment:
- [ ] When I start your application, it directs me to the error page with a URL in this format:
`localhost/appname`. This is caused by the `homepage` property that you added to your package.json file. Kindly remove that property so that the application starts with the Home page as specified in your `BrowserRouter` component of react. 👍 💻 

#### Error Page 

![image](https://user-images.githubusercontent.com/77106962/235765531-701d1379-be92-40fd-9b6d-d33f60c68071.png)

Reply:
Okay, I will do it. I added the homepage after I created the pull request. I appreciate your constructive feedback.

Reply:
@Lul-Abdifan, you are welcome.